mlrun/api/api/endpoints/tags.py

```diff
@@ -25,10 +25,10 @@
 import mlrun.common.schemas
 from mlrun.utils.helpers import tag_name_regex_as_string
 
-router = fastapi.APIRouter()
+router = fastapi.APIRouter(prefix="/projects/{project}/tags/{tag}")
```

Comment:
Tags currently support detail endpoints but that is not true for the future as each endpoint (tags in this case) need to support list endpoints as well

tl;dr - when you use router prefix, do it for the restful bucket not containing its ID too